google-cloud-pubsub/src/test/java/com/google/cloud/pubsub/v1/PublisherImplTest.java

```diff
@@ -384,41 +332,6 @@ public void testPublishFailureRetries_maxRetriesSetUnlimited() throws Exception
     publisher.shutdown();
   }
 
-  public void testPublishFailureRetries_exceededsRetryDuration() throws Exception {
```

Comment:
This test has been disabled for a long time. It tests retries, which is already tested in GAX so we can remove this now.